tests/unit/AnalyticsTest.php

```diff
@@ -25,17 +25,25 @@
  */
 use PHPUnit\Framework\TestCase;
 use PrestaShop\Module\AutoUpgrade\Analytics;
-use PrestaShop\Module\AutoUpgrade\Parameters\FileConfigurationStorage;
+use PrestaShop\Module\AutoUpgrade\Parameters\FileStorage;
 use PrestaShop\Module\AutoUpgrade\Parameters\UpgradeConfiguration;
 use PrestaShop\Module\AutoUpgrade\State\RestoreState;
 use PrestaShop\Module\AutoUpgrade\State\UpdateState;
+use PrestaShop\Module\AutoUpgrade\UpgradeContainer;
 
 class AnalyticsTest extends TestCase
 {
+    protected function setUp()
+    {
+        parent::setUp();
+        $this->container = new UpgradeContainer(__DIR__, __DIR__ . '/..');     // We expect in these tests to NOT update the theme
```

Comment:
```suggestion
        $this->container = new UpgradeContainer(__DIR__, __DIR__ . '/..');
```

The comment is wrong in this file, and irrelevant based on the option that is now deprecated.